watertap/unit_models/MD/membrane_distillation_base.py

```diff
@@ -280,7 +463,7 @@ def _add_mass_flux(self):
             self.flowsheet().config.time,
             self.difference_elements,
             initialize=1e3,
-            bounds=(1e-10, 1e5),
+            bounds=(1e-10, 1e10),
```

Comment:
Did you try remove bounds all together? If it didn't mess up solver convergence, I'd probably lean towards omitting bounds.

Reply:
I don't recall exactly whether I tried removing the bounds at any point.